client/servers/manager_test.go

```diff
@@ -53,126 +53,95 @@ func testManagerFailProb(t *testing.T, failPct float64) (m *servers.Manager) {
 func TestServers_SetServers(t *testing.T) {
 	ci.Parallel(t)
 
-	require := require.New(t)
 	m := testManager(t)
-	var num int
-	num = m.NumServers()
-	if num != 0 {
-		t.Fatalf("Expected zero servers to start")
-	}
+	must.Zero(t, m.NumServers())
 
 	s1 := &servers.Server{Addr: &fauxAddr{"server1"}}
 	s2 := &servers.Server{Addr: &fauxAddr{"server2"}}
-	require.True(m.SetServers([]*servers.Server{s1, s2}))
-	require.False(m.SetServers([]*servers.Server{s1, s2}))
-	require.False(m.SetServers([]*servers.Server{s2, s1}))
-	require.Equal(2, m.NumServers())
-	require.Len(m.GetServers(), 2)
+	must.True(t, m.SetServers([]*servers.Server{s1, s2}))
+	must.False(t, m.SetServers([]*servers.Server{s1, s2}))
+	must.False(t, m.SetServers([]*servers.Server{s2, s1}))
+	must.Eq(t, 2, m.NumServers())
+	must.Len(t, 2, m.GetServers())
 
-	require.True(m.SetServers([]*servers.Server{s1}))
-	require.Equal(1, m.NumServers())
-	require.Len(m.GetServers(), 1)
+	must.True(t, m.SetServers([]*servers.Server{s1}))
+	must.Eq(t, 1, m.NumServers())
+	must.Len(t, 1, m.GetServers())
 
 	// Test that the list of servers does not get shuffled
 	// as a side effect when incoming list is equal
-	require.True(m.SetServers([]*servers.Server{s1, s2}))
+	must.True(t, m.SetServers([]*servers.Server{s1, s2}))
 	before := m.GetServers()
-	require.False(m.SetServers([]*servers.Server{s1, s2}))
+	must.False(t, m.SetServers([]*servers.Server{s1, s2}))
 	after := m.GetServers()
-	require.Equal(before, after)
+	must.Eq(t, before, after)
```

Comment:
Below we're using `must.Equal` instead of `must.Eq`, which means this object must have a `Equal` method. Should we be using that here instead?